docs/team/member-guide.md

```diff
@@ -0,0 +1,101 @@
+# JupyterHub team guide
```

Comment:
I haven't looked at the rendered version - but the mix of .md and .rst is presumably okay?

Reply:
ya, we have the RecommonMark processor working: https://122-113493831-gh.circle-artifacts.com/0/html/team/adding-members.html

Reply:
:+1:

Reply:
Because rst by itself isn't confusing enough we let people use md sometimes but not always  :-p gotta keep people on their toes!